content/general/limits.textile

```diff
@@ -0,0 +1,306 @@
+---
+title: Limits
+section: general
+index: 20
+meta_description: "Limits are the maximum rates for operations in your account."
```

Comment:
Is "rates" the right term for this? Limits come in different flavours - some of them are related to rates, other to pure count of some entities, so I am not sure if this is correct.

Reply:
> Is "rates" the right term for this? Limits come in different flavours - some of them are related to rates, other to pure count of some entities, so I am not sure if this is correct.

Maybe: "The limits that apply to operations on your account"?

Reply:
Sounds good to me 👍

Reply:
Yes, good point - updated it thank you both.